src/mercury_engine_data_structures/formats/bmtun.py

```diff
@@ -48,3 +49,15 @@ class Bmtun(BaseResource):
     @functools.lru_cache
     def construct_class(cls, target_game: Game) -> Construct:
         return BMTUN
+
+    def get_tunable(self, tunable: str, param: str) -> Container:
```

Comment:
I don't like that this returns the container. it should just return the value directly

Reply:
Value of the param or the tunable?

Reply:
`self.raw.classes[tunable].tunables[param].value`